<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>272</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sensor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="KFilterProxySearchLine" name="searchBox"/>
      </item>
      <item>
       <widget class="QListView" name="sensorView">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KIntNumInput" name="updateSpeed">
     <property name="label">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="prefix">
      <string>Update every </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="animateBubbles">
     <property name="text">
      <string comment="@option:check Whether the plasmoid is animated or not">Animated</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showText">
     <property name="text">
      <string>Show Text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
